### PR TITLE
Bugfix/co 199

### PR DESCRIPTION
**Pull request by DanielVelando**

### CODE REVIEW

File: frontend/src/pages/mytruck/index.tsx [hunk 1]
```diff
@@ -107,6 +116,11 @@ const MyTruck: NextPage<Props> = ({ truckInfo, coupons, viewOnly = false }) => {
     [awaitCallback, refillingLocation]
   );
 
+  useEffectAsync(async () => {
+    const truck: TruckInfoDetailsDto = (await getTruck(truckInfo.id)).toJSON();
```
Comment (objmj-itminds) [inline frontend/src/pages/mytruck/index.tsx:120]: dont use `toJSON()` here

File: frontend/src/pages/mytruck/index.tsx [hunk 2]
```diff
@@ -228,7 +245,7 @@ export const getServerSideProps: GetServerSideProps<Props & I18nProps<Locale>> =
 
   const truckClient = await genTruckClient();
 
-  const truckInfo = await truckClient.getTruck(me.truckId).then(x => x.toJSON());
+  const truckInfo = (await getTruck(me.truckId)).toJSON(); // await truckClient.getTruck(me.truckId).then(x => x.toJSON());
```
Comment (objmj-itminds) [inline frontend/src/pages/mytruck/index.tsx:248]: I don't like wrapping prose chains like this.
Please do
` await getTruck(me.truckId).then(x => x.toJSON());

Comment (DanielVelando) [reply frontend/src/pages/mytruck/index.tsx:248]: That was a mistake, forgot to roll back changes!

File: frontend/src/pages/mytruck/index.tsx (hunk 1 above)
Comment (DanielVelando) [reply frontend/src/pages/mytruck/index.tsx]: +1

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**